etc/zones.conf

```diff
@@ -60,8 +60,7 @@ zone.external.force_shutdown_policy = 10000|32MB
 ## Depth so big may lead to subscribing performance issues.
 ##
 ## Value: Number [0-65535]
-## Default 7
-zone.external.max_topic_levels = 7
+## zone.external.max_topic_levels = 7
```

Comment:
Was the value 7 released?

Reply:
not yet, changed after tag [e4.2.9](https://github.com/emqx/emqx/releases/tag/e4.2.9)
the commit: [e57c30a](https://github.com/emqx/emqx/commit/e57c30a0b9699807a8c14f781ef1f09e01e41b1d)